src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/Program.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This functional test should be added in `Wasm.Build.Tests` too, so we can test it against EMSDK, and workloads.

Reply:
I took a look at the `Wasm.Build.Tests`. There is only one project file. I could add a runtimeconfig.template.json file there. Then add a test case in one or some of the test files to check if it gets picked up properly. Or do you think it is better to add a brand new test file for this?

Reply:
Created an issue for this task. https://github.com/dotnet/runtime/issues/56799